### PR TITLE
Fixes podman pod create --pod-id-file #6292

### PR DESCRIPTION
**Pull request by sujil02**
prints pod id pod create --pod-id-file and adds relevant test cases

Signed-off-by: Sujil02 <sushah@redhat.com>

### CODE REVIEW

File: test/e2e/pod_create_test.go [hunk 1]
```diff
@@ -284,4 +286,26 @@ var _ = Describe("Podman pod create", func() {
 		podCreate.WaitWithDefaultTimeout()
 		Expect(podCreate.ExitCode()).To(Equal(125))
 	})
+
+	FIt("podman create pod and print id to external file", func() {
```
Comment (rhatdan) [inline test/e2e/pod_create_test.go:290]: This should be It, not Fit.

Comment (sujil02) [reply test/e2e/pod_create_test.go]: Fixed.

**Comment by rhatdan** [comment]
LGTM

File: cmd/podman/pods/create.go [hunk 2]
```diff
@@ -147,6 +148,12 @@ func create(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Println(response.Id)
+	if len(podIDFile) > 0 {
+		if err = ioutil.WriteFile(podIDFile, []byte(response.Id), 0644); err != nil {
+			return errors.Wrapf(err, "failed to write pod ID to file %q", podIDFile)
+		}
+	} else {
```
Comment (mheon) [inline cmd/podman/pods/create.go:155]: Should this be an else? I think we always print response.Id even if we have a pod ID file - that's how `--cidfile` works

Comment (sujil02) [reply cmd/podman/pods/create.go]: Fixed.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by edsantiago** [comment]
Wish I'd seen this. @sujil02 could you open a new PR to enable the commented-out BATS test?

**Pull request merged**